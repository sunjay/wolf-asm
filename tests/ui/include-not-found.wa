.include 'does-not-exist.wa'
