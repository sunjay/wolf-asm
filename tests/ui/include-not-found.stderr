[../tests/ui/include-not-found.wa:1:10-1:28] error: unable to read included source file: `../tests/ui/does-not-exist.wa`: No such file or directory (os error 2)

error: aborting due to 1 previous error

